app/routes/groups_.new.tsx

```diff
@@ -45,11 +46,17 @@ export async function action({ request }: ActionFunctionArgs) {
     groupImageUrl = `https://fzzehiiwadkmbvpouotf.supabase.co/storage/v1/object/public/group-cover-images/${data.path}`;
   }
 
-  // TODO generate alt text or prompt user for alt text
   const new_group = await db.group.create({
-    data: { name, description, imgUrl: groupImageUrl, imgAlt: 'Group image' },
+    data: { name, description, imgUrl: groupImageUrl, imgAlt: 'The group image' },
```

Comment:
Would placing a group name by default instead of 'The group image' be better?

Reply:
Not sure! The name of the group will most likely already be a heading on the page, so I believe adding it to the image alt text group be repetitive?